BDArmory/Control/MissileFire.cs

```diff
@@ -5252,9 +5252,11 @@ bool SmartPickWeapon_EngagementEnvelope(TargetInfo target)
                                 {
                                     candidateTDPS += candidateDetDist; // weight selection towards misiles with proximity warheads
                                 }
-                                if (heat && heatTarget.exists && heatTarget.signalStrength < heatThresh)
+                                if (heat && heatTarget.exists && heatTarget.signalStrength *
+                                        ((BDArmorySettings.ASPECTED_IR_SEEKERS && Vector3.Dot(guardTarget.vesselTransform.up, vessel.vesselTransform.up) > 0.25f) ?
```

Comment:
Would this be better using `mlauncher.transform.forward` instead of `vessel.vesselTransform.up` in case the missile isn't mounted parallel with the vessel?